packages/cli/generators/controller/templates/src/controllers/controller-rest-template.ts.ejs

```diff
@@ -108,12 +111,19 @@ export class <%= className %>Controller {
     responses: {
       '200': {
         description: '<%= modelName %> model instance',
-        content: {'application/json': {schema: getModelSchemaRef(<%= modelName %>)}},
+        content: {
+          'application/json': {
+            schema: getModelSchemaRef(<%= modelName %>, {includeRelations: true}),
+          },
+        },
       },
     },
   })
-  async findById(@param.path.<%= idType %>('id') id: <%= idType %>): Promise<<%= modelName %>> {
-    return this.<%= repositoryNameCamel %>.findById(id);
+  async findById(
+    @param.path.<%= idType %>('id') id: <%= idType %>,
+    @param.query.object('filter', getFilterSchemaFor(<%= modelName %>)) filter?: Filter<<%= modelName %>>
```

Comment:
Do we support all filter properties in `findById`? For example, I find it weird to combine `findById` with `where` or `limit`/`skip` filter properties, because `findById` always returns a single record matching the given id only. Should we perhaps describe only `filter.include` and `filter.fields` properties in the schema?

Reply:
See also https://github.com/strongloop/loopback-next/issues/3336 _Filter on findById is ignoring the where clause_.

I am concerned that if we describe `filter` parameter of `findById` method as accepting all filter properties, then our users will try to set those (unsupported) properties and get surprised/confused.

Reply:
@bajtos I think `findById` only works for inclusion atm.

And I agree that the filter for `findById` should work for `inclusion` and `field` as you described in https://github.com/strongloop/loopback-next/issues/1721. Order, limit, where, etc. don't quite make sense to me. 

I'd like to land this PR first, then we can limit it in #1721 to fix #3336. 

I will try to document this issue in our doc ( I still couldn't find a best place for it) when I update inclusion docs.